lib/csl/role_mapper.rb

```diff
@@ -0,0 +1,110 @@
+module Csl
+
+  class RoleMapper
+
+    class << self
+
+      # Parse authors
+      def authors(pub_hash)
+        return [] unless pub_hash[:author].present?
```

Comment:
Does it make sense to do:
```ruby
return [] unless pub_hash && pub_hash[:author].present?
```
in case a `nil` value gets passed in here?  Seems preferable to raising an exception.

Reply:
I know that isn't the point of this PR, just something I noticed.

Reply:
Well, that would avoid an exception, but we're in big trouble here if the pub_hash is nil.  So maybe that _should_ raise an exception.